client.go

```diff
@@ -14,7 +14,17 @@ const (
 	// DefaultPort is the default TeamSpeak 3 ServerQuery port.
 	DefaultPort = 10011
 
+	// MaxParseTokenSize is the maximum buffer size used to parse the
+	// server responses.
+	// Its relatively large to enable us to deal with the typical responses
```

Comment:
It is or "It's" as you're using "Its" as a contraction of it is rather than denoting ownership.